### ISSUE TITLE
 Station name not being displayed on Now Playing

### ISSUE THREAD

**Issue by furacao14** [amp64/sonosbugtracker#19]
I use Amazon Music and I listen mostly to their Stations. When I play a Station, the Station name doesn't appear on Now Playing, just Song and Artist names.

**Comment by amp64** [comment]
Did this used to work? I don't see the station name in the Desktop app or my own apps.
The ios app is missing the Album name that the other apps do show.

**Comment by furacao14** [comment]
Yes, it used to work. And it still works for Playlists, but not for Stations.

**Comment by furacao14** [comment]
BTW, not sure, but this may be related to #18

**Comment by furacao14** [comment]
Just noticed this:
- If the Station is played from the Windows Desktop app, then the station name appears on Now Playing, both on the Desktop app and on the new Android app.
- Additionally, then it also appears on Recently Played on both platforms, so this issue is definitely related to #18.
- Playing the station again, from the Recently Played on Desktop app works, but creates a duplicate item on Recently Played on the new app, but not on the Desktop app.
- Playing the station again, from the Recently Played on the new app doesn't work and displays the "Something went wrong" error message.
